worlds/stardew_valley/logic/grind_logic.py

```diff
@@ -21,9 +22,14 @@
 else:
     ToolLogicMixin = object
 
-MIN_ITEMS = 10
-MAX_ITEMS = 999
-PERCENT_REQUIRED_FOR_MAX_ITEM = 24
+MIN_MEDIUM_ITEMS = 10
+MAX_MEDIUM_ITEMS = 999
+PERCENT_REQUIRED_FOR_MAX_MEDIUM_ITEM = 24
+
+EASY_ITEMS = {Material.wood, Material.stone, Material.fiber, Material.sap}
+MIN_EASY_ITEMS = 500
```

Comment:
I'd set 100 here, instead of 500. Maybe 200. My problem with 500 is the sticky bundle, which this whole grinding logic concept was originally invented to push out of sphere 1

Reply:
This would impact the rules for most blueprints, fine with that?

<details><summary>Before</summary>
<p>

![image](https://github.com/user-attachments/assets/a042c5eb-115d-41aa-928a-02ff21c867ec)

</p>
</details> 


<details><summary>After with 200</summary>
<p>

![image](https://github.com/user-attachments/assets/2e47ac68-700f-4d4d-b8bd-e95c33838bb1)

</p>
</details>

Reply:
Honestly that seems valid to me

Reply:
As discussed, I've set it to 300 so the beach bridge stay in sphere 1.